src/sections/Meshery/Features-Col/data/index.js

```diff
@@ -7,7 +7,7 @@ const data = {
       btnText: "Get Started",
       services: [
         {
-          content: "AWS App Mesh, Citrix Service Mesh, Consul, Istio, Kuma, Linkerd, Network Service Mesh, NGINX Service Mesh, Octarine, Open Service Mesh, Traefik Mesh",
+          content: "AWS App Mesh, Citrix Service Mesh, Consul, Istio, Kuma, Linkerd, Network Service Mesh, NGINX Service Mesh, Octarine, Open Service Mesh, Traefik Mesh, Cilium Service Mesh",
```

Comment:
The rest of the list is alphabetical. Please fix.

Reply:
ok

Reply:
it is fixed now !! 

![image](https://user-images.githubusercontent.com/64387054/146455233-ac5fabc8-ca3a-462c-9185-1c0cfcf55c0c.png)